src/main.tsx

```diff
@@ -4,42 +4,50 @@ import './index.css'
 import { Provider } from 'react-redux'
 import { setupStore } from './redux/store.tsx'
 import { createBrowserRouter, RouterProvider } from 'react-router-dom'
-import { Outlet } from 'react-router-dom'
+import WorldInformation from './components/WorldInformation.tsx'
+
+
+//import { Outlet } from 'react-router-dom'
 import Navbar from './components/Navbar'
 import Consult from './components/Consult.tsx'
 const NavbarWrapper = () => {
   return (
   <div>
       <Navbar/>
-      <Outlet/>
+      <WorldInformation/>
   </div>
   )
-};
+}; 
+
+
 const router = createBrowserRouter([
   {
     path: "/", 
-    element: <NavbarWrapper/>,
+    element:
+    <NavbarWrapper />,
+
+    
     children: [
-      {
-      path: "/",
-      element: (
-        <h1>Home</h1>
```

Comment:
# Great job!!! 
The information is recovered correctly and displayed very well! I suggest add your `<WorldInformation/>` component to this place.  :grinning: